haiku-apps/trax/trax-1.1.1.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Find files by name AND location, unlike Haiku's Find"
+DESCRIPTION="TraX is not a \"query\" tool. It does not keep a database of files and attributes. \
+It is a traditional find tool, which searches the disk every time, looking for files that meet \
+your criteria. The find-criteria are not Haiku specific, and it does not (currently) search \
+attributes.
+
+TraX comes in two varieties.
+
+ * FastTraX is similar to the Windows find tool. Windows users should find its features very \
+familiar.
+ * TraX is a cross between the UNIX \"find\" command the the MacOS \"Find Files\" tool. It has \
+much of the power of \"find\" with a Mac-like interface (a Mac on steroids that is)."
+HOMEPAGE="https://github.com/HaikuArchives/TraX/"
+COPYRIGHT="1999-2000, Tim Vernum"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="ed09bd45e6cd9faa365c8c1f542f1636aa19cd63"
+SOURCE_URI="https://github.com/HaikuArchives/TraX/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="8a4261a787fa861ee40b78448e73e00e50800d2b660945c813a14ca5df31c8b4"
+SOURCE_DIR="TraX-$srcGitRev"
+
+ARCHITECTURES="all"
+
+PROVIDES="
+	trax = $portVersion
+	app:FastTraX = $portVersion
+	app:TraX = $portVersion
+	lib:libfind = $portVersion
+	lib:libTraX = $portVersion
```

Comment:
Should the libraries be versioned at all? They don't really have a version.

If they aren't installed in `$libDir`, they also don't need to be listed here.

Reply:
If you version them then you should start the `$libVersion` at 0.0.0.  There are a number of recipes that do this.

Reply:
The HaikuArchives repo contains a resource file for libfind.so with the version number 0.0.0, but it isn't actually used. So effectively, both these libraries have no version at all.

Reply:
I removed the version numbers.